documentation/source/building.rst

```diff
@@ -51,6 +51,12 @@ Make Variables
     ``EXTRA_ARGS``
       Passed to both the compile and execute phases of simulators with two rules, or passed to the single compile and run command for simulators which don't have a distinct compilation stage.
 
+    ``COCOTB_HDL_TIMESTEP``
+      The default timestep that should be assumed for simulation when not specified by modules in the design. If this isn't specified then it is assumed to be 1ns.
+
+    ``COCOTB_HDL_TIMEPRECISION``
+      The default precision that should be assumed for simulation when not specified by modules in the design. If this isn't specified then it is assumed to be 1ps.
```

Comment:
I would say "The default time precision" so that it can't be mistaken for numerical precision.